tomviz/MainWindow.ui

```diff
@@ -160,7 +167,7 @@
         <bool>true</bool>
        </property>
        <property name="columnCount">
-        <number>2</number>
+        <number>1</number>
```

Comment:
I don't know.  When I set up my new machine I made sure to install only the Qt Designer that came with Qt 4.  It says version 4.8.6.  I really don't know why it keeps changing this.